Documentation/ceph-filesystem.md

```diff
@@ -65,12 +72,71 @@ $ ceph status
     mds: myfs-1/1/1 up {[myfs:0]=mzw58b=up:active}, 1 up:standby-replay
 ```
 
+## Provision Storage
+
+Before Rook can start provisioning storage, a StorageClass needs to be created based on the filesystem. This is needed for Kubernetes to interoperate
+with the CSI driver to create persistent volumes.
+
+**NOTE** This example uses the CSI driver, which is the preferred driver going forward. Examples are found in the [CSI CephFS](https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/cephfs) directory. For an example of a volume using the flex driver, see the [Flex Driver](#flex-driver) section below.
+
+Save this storage class definition as `storageclass.yaml`:
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-cephfs
+provisioner: cephfs.csi.ceph.com
+parameters:
+  # clusterID is the namespace where operator is deployed.
+  clusterID: rook-ceph
+
+  # CephFS filesystem name into which the volume shall be created
+  fsName: myfs
+
+  # Ceph pool into which the volume shall be created
+  # Required for provisionVolume: "true"
+  pool: myfs-data0
+
+  # Root path of an existing CephFS volume
+  # Required for provisionVolume: "false"
+  # rootPath: /absolute/path
+
+  # The secrets contain Ceph admin credentials. These are generated automatically by the operator
+  # in the same namespace as the cluster.
+  csi.storage.k8s.io/provisioner-secret-name: rook-ceph-csi
+  csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph
+  csi.storage.k8s.io/node-stage-secret-name: rook-ceph-csi
+  csi.storage.k8s.io/node-stage-secret-namespace: rook-ceph
+
+reclaimPolicy: Delete
+mountOptions:
+  - noexec
```

Comment:
@travisn I put that option in my patch and forgot to change here. Lets mention `debug` can as the mount option here.

Reply:
instead of `noexec` put `debug`

Reply:
Ok i'll add `debug` and remove `noexec`. What is the behavior of the debug option? More logging?
```
mountOptions:
  - debug
```